function-images/tests/chained-function-serving/consumer/consumer.go

```diff
@@ -42,6 +46,13 @@ type consumerServer struct {
 
 func (s *consumerServer) ConsumeString(ctx context.Context, str *pb.ConsumeStringRequest) (*pb.ConsumeStringReply, error) {
 	log.Printf("[consumer] Consumed %v\n", str.Value)
+	ctx, end := tracing.Span(ctx, "my_span", "tracer")
+	defer end()
+	<-time.After(1 * time.Second)
+	ctx, childEnd := tracing.Span(ctx, "my_sub_span", "tracer")
+	<-time.After(1 * time.Second)
+	childEnd()
+	<-time.After(1 * time.Second)
```

Comment:
this is to be removed from the bench. Please have the corresponding unit test in the tracing module